packages/ag-charts-community/src/chart/label.ts

```diff
@@ -145,3 +233,23 @@ export function createLabelData(
 
     return labelData;
 }
+
+export function createFixedLabelData(
+    { width, height, spacing }: { width: number; height: number; spacing: number },
+    labelX: number,
+    labelMatrix: Matrix
+): PlacedLabelDatum[] {
+    const bbox1 = labelMatrix.transformBBox(new BBox(labelX, 0, 0, 0), new BBox(labelX, 0, width, height));
+    const bbox2 = labelMatrix.transformBBox(new BBox(labelX, spacing, 0, 0), new BBox(labelX, spacing, width, height));
```

Comment:
```suggestion
    const bbox1 = labelMatrix.transformBBox(new BBox(labelX, 0, 0, 0));
    const bbox2 = labelMatrix.transformBBox(new BBox(labelX, spacing, 0, 0));
```
nit: why initialise the `target` BBox with arguments if it's going to be overwritten anyhow?
I think the `target` param optimisation is to skip creating new BBox instances, but if we're doing it anyhow I think it's redundant.